napari/plugins/hub.py

```diff
@@ -44,6 +44,18 @@ def hub_plugin_info(
     except error.HTTPError:
         return None, False
 
+    # If the napari hub returns an empty info dict,
+    # simply return None, False like the above except
+    if info == {
+        'conda': [],
+        'display_name': '',
+        'plugin_types': [],
+        'reader_file_extensions': [],
+        'writer_file_extensions': [],
+        'writer_save_layers': [],
+    }:
```

Comment:
```suggestion
    if 'name' not in info:
```

Reply:
just to be safe, should it maybe be everything that gets directly indexed?
`'name', 'version', 'authors', 'summary', 'license'`?  or is there no chance that if name makes it in, the other ones don't make it in?